reference-architecture/aws-ansible/playbooks/roles/prerequisites/tasks/main.yaml

```diff
@@ -11,3 +11,4 @@
       - bash-completion
       - docker
       - python-httplib2
+  when: ansible_pkg_mgr != "unknown" 
```

Comment:
Ah, we probably need to make sure that there is a default ansible fact that will expose an Atomic Host.

Until that is available, you can test for `/run/ostree-booted` or use the openshift.common.is_atomic fact from openshift_facts.